src/java.base/share/classes/java/util/regex/Pattern.java

```diff
@@ -4006,7 +4006,9 @@ boolean match(Matcher matcher, int i, CharSequence seq) {
                 if (j < matcher.to)
                     return false;
             }
-            matcher.hitEnd = true;
+            else {
```

Comment:
Is there a reason the "else" starts on its own line?

Reply:
I must love Enter too much. Thanks for the catch!